src/api/app/helpers/webui/notification_helper.rb

```diff
@@ -37,4 +131,11 @@ def notification_filter_active?(filter_item)
   def badge_color(filter_item)
     notification_filter_active?(filter_item) ? 'badge-light' : 'badge-primary'
   end
+
+  def reviewers_tags(review)
+    return user_avatar(User.find_by(login: review.by_user)) if review.by_user
+    return user_avatar(Group.find_by(title: review.by_group)) if review.by_group
+    return simulated_avatar('fa-archive text-warning', "Package #{review.by_project}/#{review.by_package}") if review.by_package
+    return simulated_avatar('fa-cubes text-secondary', "Project #{review.by_project}") if review.by_project
+  end
```

Comment:
All the methods in this helper that use `capture` are partials. Is there a reason we using helper methods instead?

Reply:
The complex part here is displaying as "avatars" many different things: sometimes real avatars, sometimes simple icons and sometimes  the counter (`+N`). I need the helpers you mention because the are called from many places. 

Apart from that, I can't convert them to partials because I need to call them inside the `request_tags` helper. Which is the one that gives us an array of things we can really use.

Reply:
I'm not talking only about the helper above. I'm talking about all the helpers that use some form of `capture`,

Reply:
We have moved as much as we could to the view. Now can be easier to understand.